src/bbs2gh/Commands/GenerateScriptCommand.cs

```diff
@@ -65,6 +68,16 @@ public class GenerateScriptCommand : CommandBase<GenerateScriptCommandArgs, Gene
         description: "The SSH port (default: 22).",
         getDefaultValue: () => 22);
 
+    public Option<string> SmbUser { get; } = new(
+        name: "--smb-user",
+        description: "The SMB user to be used for downloading the export archive off of the Bitbucket server." +
+                     $"{Environment.NewLine}" +
+                     "Note: The SMB password will also be required in order to connect to a SMB share which has to be set as an environment variable before running the script.");
+
+    public Option<string> Domain { get; } = new(
+        name: "--domain",
+        description: "The optional domain name when using SMB for downloading the export archive.");
```

Comment:
How does the user know if they need to specify a domain name? Will it be obvious to them? I don't know too much about Windows server administration.

Reply:
I never actually tested the SMB download with a domain, I guess if there is a domain then they probably need to specify it.